tests/test_local_sampling_benchmark.py

```diff
@@ -0,0 +1,16 @@
+import json
+import pkgutil
+from pkg_resources import resource_filename
+
+from aws_xray_sdk.core.sampling.local.sampler import LocalSampler
+
+def test_pkg_resources_static_read(benchmark):
+    def get_sampling_rule():
+        with open(resource_filename(__name__, 'mock_sampling_rule.json')) as f:
+            return json.load(f)
+    benchmark(get_sampling_rule)
+
+def test_pkgutil_static_read(benchmark):
+    def get_sampling_rule():
+        json.loads(pkgutil.get_data(__name__, 'mock_sampling_rule.json'))
+    benchmark(get_sampling_rule)
```

Comment:
These tests are really testing the performance of standard library functions (`pkgutil` and `pkg_resources`) instead of testing `aws-xray-sdk-python` functions.

For that reason I think we should not include them here? The PR comment can serve as proof that this change is worth it.

Otherwise, we can leave the tests here so the results can be reproduced later.

Reply:
Agree with deleting these benchmarks (first thing that came to mind but keeping them also isn't a big deal)